packages/gatsby-source-wordpress/README.md

```diff
@@ -83,11 +83,19 @@ plugins: [
       // Set verboseOutput to true to display a verbose output on `npm run develop` or `npm run build`
       // It can help you debug specific API Endpoints problems
       verboseOutput: false,
+      // Set how many pages are retrieved per API request
+      perPage: 100,
       // Search and Replace Urls across WordPress content
       searchAndReplaceContentUrls: {
         sourceUrl: "https://source-url.com",
         replacementUrl: "https://replacement-url.com",
       },
+      // Exclude specific endpoint manufacturers
+      // Example: [`yoast`] will exclude all `/yoast/` endpoints such as `/yoast/v1/statistics` from fetch.
+      // excludedManufacturers: [`disqus`, `yoast`],
+      // Exclude sepcific endpoint types
+      // Example: [`comments`] will exclude all `comments` type endpoints such as `/wp/v2/comments` from fetch.
+      // excludedTypes: [`comments`, `media`],
```

Comment:
I feel like splitting exclusion into `excludedManufacturers` and `excludedTypes` limit flexibility of this feature. I can't really exclude `/acf/v3/posts` with this approach (either I exclude `acf` manufacturer and lose access to acf pages route or I exclude `posts` type and lose `/wp/v2/posts`). 

I would merge those into single `excludedRoutes` or `excludedEndpoints` and allow using globs (with [minimatch](https://www.npmjs.com/package/minimatch) package) or regex there so I could define them like this:
`excludedRoutes: ['acf/posts', '*/comments', 'yoast/*']`

Thoughts?

Also uncomment lines where you show how to exclude things. We want actual options to be quickly visible and now they blend with comments.

Reply:
I agree this is a more flexible approach, I'll update shortly.